feature-libs/user/profile/occ/adapters/occ-user-profile.adapter.ts

```diff
@@ -51,6 +54,12 @@ export class OccUserProfileAdapter implements UserProfileAdapter {
       'Content-Type': 'application/json',
     });
     headers = InterceptorUtil.createHeader(USE_CLIENT_TOKEN, true, headers);
+    if (this.captchaService.getToken()) {
```

Comment:
Since this logic is repeated, can we create a function for it? eg. `headers = appendCaptchaToken()`